src/Compilers/VisualBasic/Portable/GlobalImport.ImportDiagnosticInfo.vb

```diff
@@ -52,6 +52,26 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 _startIndex = startIndex
                 _length = length
             End Sub
+
+            Public Overrides Function Equals(obj As Object) As Boolean
+                If obj.GetType() Is GetType(ImportDiagnosticInfo) Then
```

Comment:
>If obj.GetType() Is GetType(ImportDiagnosticInfo) Then [](start = 16, length = 54)

This is a null reference dereference.

```
Dim other = TryCast(obj, ImportDiagnosticInfo)
If other IsNot Nothing
    use other here
End If
```

If you only want to check the type and don't need to converted object, which is not the case here, there is a TypeOf operator, i.e
```
If TypeOf obj Is ImportDiagnosticInfo
```